### ISSUE TITLE
Crossposting to PDS not supported

### ISSUE THREAD

**Issue by TheOpponent** [59de44955ebd/twitter-to-bsky#9]
Attempting to crosspost to an AT protocol PDS instance federated to the Bluesky network as of June 4, 2024 results in an "undefined" message with no further error messaging.

**Comment by 59de44955ebd** [comment]
Good point! But as I'm still only a occasional Bluesky user, I currently don't have the resources to fix this. So please go ahead, fork and fix the issue, and then make a pull request. Thanks!
